src/wallet/wallet.h

```diff
@@ -351,6 +351,11 @@ class CMerkleTx
      */
     int nIndex;
 
+    /* Refers to height of block against which tx is merkle branch linked to
+     * An block_height == 0 means that tx isn't yet linked to any block
```

Comment:
The comment says `An block_height == 0` in commit _Add m_block_height field in CMerkleTx_ and then changes to `A block_height == -1` in _Use CWallet::m_last_block_processed_height in GetDepthInMainChain_.

It would be better to not change this in the course of the PR and just set it to `A block_height == -1` in the first PR.